app/helpers/codelist_helper.rb

```diff
@@ -1,13 +1,13 @@
 # frozen_string_literal: true
 
-module OrganisationHelper
-  def yaml_to_options(type)
-    yaml = load_yaml(type)
+module CodelistHelper
+  def yaml_to_options(entity, type)
+    yaml = load_yaml(entity, type)
     data = yaml["data"]
     data.collect { |item| [item["name"], item["code"]] }
   end
 
-  def load_yaml(type)
-    YAML.safe_load(File.read("#{Rails.root}/vendor/data/codelists/IATI/#{IATI_VERSION}/organisation/#{type}.yml"))
+  def load_yaml(entity, type)
```

Comment:
👍

Reply:
This is also a really well focused commit, perfect.

Reply:
Thanks!